package.json

```diff
@@ -37,6 +37,7 @@
   "homepage": "https://github.com/babel/broccoli-babel-transpiler",
   "dependencies": {
     "@babel/core": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
```

Comment:
Should we bump/set the minimum versions of both @babel packages here? Seems fine for a follow-up PR, just a thought I had as reviewing...

Reply:
Would bumping the version to, e.g. `^7.2.0` have any detrimental effects? If not I'd just go for it.